pycascadia/utility.py

```diff
@@ -11,6 +11,12 @@ def is_region_valid(region):
     else:
         return True
 
+def all_values_are_nodata(grid):
+    if (grid.nodatavals == grid.values).all():
+        return True
+    else:
+        return False
```

Comment:
Seems cleaner to return the condition directly.

```suggestion
    return (grid.nodatavals == grid.values).all()
```